src/io/sampling.jl

```diff
@@ -6,6 +6,8 @@ import Surrogates: sample
 import Surrogates.QuasiMonteCarlo: SobolSample
 
 
+const DISCRETE_FACTOR_TYPES = ["integer", "categorical"]
```

Comment:
I think move to `Ecosystem.jl` which defines the abstract `EcoModel` type which all components are a subtype of.

`EcoModel` is a horrible name, we'll change it later.